labkey-ui-ehr/src/theme/index.scss

```diff
@@ -0,0 +1,2 @@
+@import "@labkey/components-scss";
+//@import "@labkey/components-app-scss/variables";
```

Comment:
looking at the labkey-ui-premium equivalent of this file, I think you want the commented out line here and not the other one. Right?